sebs/utils.py

```diff
@@ -251,6 +252,9 @@ def logging_handlers(self, handlers: LoggingHandlers):
 def has_platform(name: str) -> bool:
     return os.environ.get(f"SEBS_WITH_{name.upper()}", "False").lower() == "true"
 
+# Check if the system is Linux and that it's not WSL
+def is_linux() -> bool:
+    return platform.system() == "Linux" and "microsoft" not in platform.release().lower()
```

Comment:
Now I see :-) Can we use that in `sebs/storage/minio.py`?

Reply:
👍